index.js

```diff
@@ -44,8 +44,7 @@ app.use(passport.initialize());
 app.use(passport.session());
 app.use('/api/auth', authRoutes);
 app.use("/api/projects", projectRoutes);
-app.use(middleware)
```

Comment:
This line will affect the frontend routes as well. I'm not sure that's the intention, so I moved it to be namespace specific.